core/tests/webdriverio_desktop/contributorAdminDashboard.js

```diff
@@ -0,0 +1,351 @@
+    await topicsAndSkillsDashboardPage.waitForTopicsToLoad();
+    await topicsAndSkillsDashboardPage.navigateToTopicWithIndex(0);
+    await topicEditorPage.createStory(
+      'Story Title', 'topicandstoryeditorone', 'Story description',
+      Constants.TEST_SVG_PATH);
+    await storyEditorPage.createNewChapter(
+      'Chapter 1', dummyExplorationId, Constants.TEST_SVG_PATH);
+    await storyEditorPage.updateMetaTagContent('story meta tag');
+    await storyEditorPage.saveStory('Saving Story');
+    await storyEditorPage.publishStory();
+
+    // Testing the copy tool.
+    let opportunityActionButtonCss = $(
+      '.e2e-test-opportunity-list-item-button');
+    let copyButton = $('.e2e-test-copy-button');
+    let doneButton = $('.e2e-test-close-rich-text-component-editor');
+    let saveButton = $('.e2e-test-save-button');
+    let textbox = $('.e2e-test-description-box');
+
+    await contributorDashboardPage.get();
+    await waitFor.pageToFullyLoad();
+    await contributorDashboardPage.navigateToTranslateTextTab();
+    await contributorDashboardTranslateTextTab.changeLanguage(
+      'shqip (Albanian)');
+    await contributorDashboardPage.waitForOpportunitiesToLoad();
+    await action.click('Opportunity button', opportunityActionButtonCss);
+    let image = $('.e2e-test-image');
+    await waitFor.visibilityOf(
+      image,
+      'Test image taking too long to appear.');
+    let images = await $$('.e2e-test-image');
+    expect(images.length).toEqual(1);
+
+    // Copy tool should copy image on pressing 'Done'.
+    await waitFor.visibilityOf(
+      copyButton, 'Copy button taking too long to appear');
+    await action.click('Copy button', copyButton);
+    await action.click('Image', images[0]);
+    await action.setValue(
+      'Set Image description', textbox, 'An example description');
+    await action.click('Done', doneButton);
+    await action.click('Save', saveButton);
+    await users.logout();
+
+    // Accept suggestion as user1.
+    await users.login(USER_EMAILS[1]);
+    await contributorDashboardPage.get();
+    await contributorDashboardPage.selectTranslationReviewTab();
+    await contributorDashboardPage.waitForOpportunitiesToLoad();
+    await contributorDashboardPage.selectReviewLanguage('shqip (Albanian)');
+
+    if (width < 700) {
+      let opportunityItemSelector = function() {
+        return $$('.e2e-test-opportunity-list-item');
+      };
+      let opportunity = (await opportunityItemSelector())[0];
+      await action.click('Opportunity Item', opportunity);
+    } else {
+      await contributorDashboardPage.clickOpportunityActionButton(
+        'Chapter 1', 'Topic 0 for contribution - Story Title');
+    }
+
+    if (width < 700) {
+      let opportunityItemSelector = function() {
+        return $$('.e2e-test-opportunity-list-item');
+      };
+      let opportunity = (await opportunityItemSelector())[0];
+      await action.click('Opportunity Item', opportunity);
```

Comment:
maybe define this in the utils file as clickFiestOpportunityInSmallScreenView?

Also, why are we selecting the random first item where as for bigger screen selecting a specific item?

Reply:
Done. PTAL.